app/scripts/injected/main.js

```diff
@@ -130,18 +130,24 @@ sap.ui.require(['ToolsAPI'], function (ToolsAPI) {
             var newValue = data.value;
 
             var control = sap.ui.getCore().byId(controlId);
-            if (control) {
-                // Change the property through its setter
-                control['set' + property](newValue);
 
-                // Update properties and bindings
+            try {
+                if (control) {
+                    // Change the property through its setter
+                    control['set' + property](newValue);
+                }
+            } catch (error) {
+                // Update properties and bindings after an error
```

Comment:
I've added this to always update the devtool with the actual value which might not be the same as handed over, e.g. "null" => property default value. So imo this should also be done when the actual update succeeds.